nexus/base/utils/dynamic_fixture_utils.py

```diff
@@ -27,3 +30,8 @@ def jsonfield_config(self, field, key):
     def filefield_config(self, field, key):
         f = File('/tmp/a')
         return f
+
+    def versatileimagefield_config(self, field, key):
+        image = create_image(None, 'image.png')
```

Comment:
Can you please use named arguments, so that we can quickly understand what is `None` for? 

Likewise everywhere else.